client/src/pages/Country/Country.tsx

```diff
@@ -169,14 +188,33 @@ export const Country: React.FC<CountryProps> = ({ country }) => {
         />
 
         {catResults.length > 0 ? (
-          <div className="resultContainer">
+          <>
+            <h3>Number of Likes vs Trending Date</h3>
+            <LineGraph
+              results={catResults
+                .map((video) => ({
+                  ...video,
+                  pub_date: video.pub_date.replace("-", "/"),
```

Comment:
Nit: Since we have changed the "-" to "/" in the server in recent changes, we don't need to replace them on the client anymore. Can change this in later PRs and merge changes here though